app/containers/Settings/settings.scss

```diff
@@ -0,0 +1,59 @@
+.settingsContainer {
```

Comment:
If you capitalize the letter of this file it should fix the tests in circleCI its also just the pattern in the app (.scss file and component file share the same name)

Reply:
Wondered why the circleCI was failing! thanks